lib/plottr_components/src/components/dashboard/account/proOnboarding/ProStep1.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'react-proptypes'
+import { t } from 'plottr_locales'
+import OnboardingStep from '../../../onboarding/OnboardingStep'
+import { StepBody, StepHeader } from '../../../onboarding/Step'
+import UnconnectedFirebaseLogin from '../../../FirebaseLogin'
+
+const ProStep1Connector = (connector) => {
+  const {
+    platform: { useSettingsInfo },
+  } = connector
+
+  const FirebaseLogin = UnconnectedFirebaseLogin(connector)
+
+  const ProStep1 = ({ nextStep }) => {
+    const [settings] = useSettingsInfo()
+    useEffect(() => {
+      if (settings.user.id) nextStep()
+    }, [settings])
+
+    const checkUserId = (uid) => {
+      if (uid) nextStep()
+    }
+
+    const showFrb = settings.user && !settings.user.id
```

Comment:
I prefer connecting components to redux and using `client.userId` from there because all of the places that we use these storage hooks have odd race conditions that I've had to debug.